src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -227,14 +227,17 @@ private void decodeOtherParameter(Position position, int id, ByteBuf buf, int le
             case 26:
             case 27:
             case 28:
-                position.set(Position.PREFIX_TEMP + (id - 24), readValue(buf, length, true) * 0.1);
+                position.set("bleTemp" + (id - 24), readValue(buf, length, true) * 0.1);
```

Comment:
I don't agree with this part. Just use standard prefix and different indexes.

Reply:
Just find out that ID's 25-28 is different in FM*640 devices.
https://wiki.teltonika-gps.com/view/FMM640_AVL_ID

What do you think will be best option for that? Remove BLE temperature implementation? 
It would be great option for teltonika device to have selection of device types, and implement different protocol for each.

Reply:
If it's not consistent between different models, we should remove it.

Reply:
also my changes to CNG, conflict's with FMx640.. so i think i will cancel my PR, and will use my fork till someone figures out how to implement different protocols.